requires-ci.txt

```diff
@@ -0,0 +1,9 @@
+dash_flow_example==0.0.5
+dash-dangerously-set-inner-html
+six
+mock
+flake8
+pylint==1.9.4;python_version<"3.7"
+pylint==2.3.1;python_version=="3.7"
+astroid==2.2.5;python_version=="3.7"
```

Comment:
didn't know you could do that, cool! Would it be better to say `>="3.7"`? NBD, I guess we'll know pretty quickly if we move to 3.8 and things break...